tests/integration/security/jwt_test.go

```diff
@@ -391,6 +396,9 @@ func TestRequestAuthentication(t *testing.T) {
 						},
 					},
 					ExpectResponseCode: response.StatusCodeOK,
+					ExpectHeaders: map[string]string{
+						authHeaderKey: "Bearer " + jwt.TokenExpired,
```

Comment:
Will we always forward the invalid token even if forwardOriginalToken is false? I guess this is because we haven't implemented the empty JWT token support and will be changed once your other PR is merged, just to confirm.

Reply:
That's right. https://github.com/istio/istio/pull/20054 will reject requests with invalid token right away.